gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -769,16 +769,19 @@ public BasicInputArea(Composite parent, Settings settings, Theme theme,
       addSeparator();
 
       bat = createCheckbox(this, "Battery", sBatt.getEnabled(), e -> updateBat());
-      batLabels = new Label[2];
       Composite batGroup = withLayoutData(
-          createComposite(this, withMargin(new GridLayout(3, false), 5, 0)),
+          createComposite(this, withMargin(new GridLayout(1, false), 5, 0)),
+          withIndents(new GridData(), GROUP_INDENT, 0));
+      batLabels = new Label[2];
+      Composite batLabelGroup = withLayoutData(
+          createComposite(batGroup, withMargin(new GridLayout(3, false), 5, 0)),
           withIndents(new GridData(), GROUP_INDENT, 0));
-      batLabels[0] = createLabel(batGroup, "Poll Rate:");
-      batRate = createSpinner(batGroup, sBatt.getRate(), 250, 60000);
-      batLabels[1] = createLabel(batGroup, "ms");
+      batLabels[0] = createLabel(batLabelGroup, "Poll Rate:");
+      batRate = createSpinner(batLabelGroup, sBatt.getRate(), 250, 60000);
+      batLabels[1] = createLabel(batLabelGroup, "ms");
       if (caps.getHasPowerRail()) {
         batPowerRail = createCheckbox(batGroup, "Collect Power Rails", sBatt.getCollectPowerRail());
-        // TODO(b/241746026): Add the checkbox for Energy Breakdown
+        batEnergyBreakDown = createCheckbox(batGroup, "Collect Energy Breakdown", sBatt.getCollectEnergyBreakdown());
```

Comment:
nit: make sure that the order of checkboxes match the order of rows in the system profile itself

Reply:
I've recently added a new version of createCheckbox function with a tooltipText, can you please add extra information about this option as a tooltip. Similarly, a tooltip can be added to 'Collect Power Rails' option, to clarify the differences between the two. Feel free to add versions of other widget creation functions with tooltips in order to make newly added UI bits more understandable for the end user.